src/viur/datastore/query.py

```diff
@@ -156,8 +156,12 @@ def mergeExternalFilter(self, filters: dict) -> 'Query':
 			logging.exception(e)
 			self.queries = None
 			return self
+		startCursor = endCursor = None
 		if "cursor" in filters and filters["cursor"] and filters["cursor"].lower() != "none":
```

Comment:
```suggestion
		if (cursor := filters.get("cursor") or "") and cursor.lower() != "none":
```

Reply:
I've done a benchmark on that two flavors of tests and also inspected the resulting bytecode. The existing code is indeed a little bit more faster than the version with the walrus operator. What I've not yet checked the resulting binary code compiled by cython.

>>> timeit(lambda: testOld(filters), number=300000000)
46.74160768299953
>>> timeit(lambda: testWalrus(filters), number=300000000)
47.252532685000915